### DISCUSSION TITLE
Can't use Openpose, preview Error

### DISCUSSION THREAD

**Discussion by CiCi1223** [Mikubill/sd-webui-controlnet#1744] [Q&A]
Unable to use openpose, error displayed during preview.
How to solve this problem？
![error](https://github.com/Mikubill/sd-webui-controlnet/assets/138381579/60813827-ab69-4e30-9322-0b5612c39eaf)

**Comment by huchenlei** [comment]
Closing for insufficient context. Please feel free to file an issue with console output and other information to reproduce the issue.
